docker-compose.yml

```diff
@@ -40,10 +40,9 @@ services:
       context: .
       dockerfile: ./docker/builder.Dockerfile
     volumes:
-      - built-static:/app/build/
-      - type: bind
-        source: ./builder/gulpfile.js
-        target: /app/gulpfile.js
+      - built-static:/home/node/app/build
+      - ./builder/src:/home/node/app/src:ro
+      - ./builder/gulpfile.js:/home/node/app/gulpfile.js:ro
```

Comment:
What's the reasoning behind the bind type change? I remember there being one specificallyf or using a bind mount but can't recall what it was exactly anymore, probably was related to how it's handled if the file is missing.

Reply:
The [docs](https://docs.docker.com/storage/bind-mounts/#choose-the--v-or---mount-flag) suggest that these definitions are equivalent so I just tried to simplify it

I can't find in the Docker compose specification any details but comparing with the [docs for volumes](https://docs.docker.com/storage/volumes/#choose-the--v-or---mount-flag), it seems only named and anonymous volumes have the type `volume`, while if you specify 2 files, the first being on the host, it is a bind volume. 

The previous way of writing it seems to just be more verbose and as there isn't necessarily anything unexpected about it, using the single line way of defining a `bind` mount should be fine and clear